.spdx-ignore

```diff
@@ -11,3 +11,5 @@ cuda_bindings/examples/*
 
 # Vendored
 cuda_core/cuda/core/experimental/include/dlpack.h
+
+qa/ctk-next.drawio.svg
```

Comment:
```suggestion
```

Reply:
Yes, okay to keep, might be better to just make it a `qa/*` wildcard? We can obviously do that in a follow up though.